Harubee/Sources/Presentation/Calendar/Views/DailyCalendarView.swift

```diff
@@ -62,6 +62,12 @@ struct DailyCalendarView: View {
             if !fixedExpenses.isEmpty {
               FixedExpenseSection(expenses: fixedExpenses)
             }
+            
+            if let selectedDate = viewModel.state.selectedDate {
+              if selectedDate.isSameDay(as: currentBudget.startDate) {
+                FixedIncomeSection(fixedIncome: currentBudget.fixedIncome)
+              }
+            }
```

Comment:
두 조건을 만족해야만 `FixedIncomeSection()`을 보여주는 것이라면 아래처럼 수정하는것이 좋아보여요!
새로운 if문을 추가하지 않고 하나의 if let 구문에 조건을 연속해서 나열해보았습니다!
```suggestion
            if let selectedDate = viewModel.state.selectedDate, 
              selectedDate.isSameDay(as: currentBudget.startDate) {
                FixedIncomeSection(fixedIncome: currentBudget.fixedIncome)
            }
```